src/pages/resources/migration/android/tabs/migrate-to-android-2x.md

```diff
@@ -0,0 +1,49 @@
+<Variant platform="java" repeat="1"/>
```

Comment:
In general we use platform=android/ios and not the programming language. If you want to show them in a tabular fashion, can we use a different variant attribute name for this, e.g. lang="java" or similar